.github/workflows/codespell_and_flake.yml

```diff
@@ -8,14 +8,28 @@ on:
       - '*'
 
 jobs:
+  check-skip:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: GuillaumeFavelier/ci-skip@enh/separator
+        with:
+          fail-fast: true
```

Comment:
This one `fail-fast` at the top of each file looks better to me than having N `if: ${{ env.CI_SKIP == 'false' }}` for our N steps in each file.

Are those the two alternatives? Or are you saying somehow `fail-fast` does not work?

Reply:
I agree it looks better. The drawback is that it doubles the number of jobs. Also if `fail-fast: true` is acceptable in the main job, we don't need to create an additional `check-skip` for that. I don't know if I'm clear so I'll push the other alternative for `codespell_and_flake` to demonstrate.